TreeTracker/UI/Views/Home/HomeViewController.swift

```diff
@@ -97,6 +103,10 @@ private extension HomeViewController {
     @IBAction func uploadsButtonPressed() {
         viewModel?.uploadListSelected()
     }
+
+    @objc func logoutButtonPressed() {
+        self.viewModel?.logoutPlanter()
```

Comment:
We don't need the `self` keyword here. I know it's valid code but I think we should be consistent throughout the codebase with this. Swift encourages us to omit `self` and only use it when the compiler requires it.